Description: Load worker MPM, to prevent error message "apache2: Configuration error: No MPM loaded.", resulting from changes in Apache 2.4.
Author: Roberto C. Sanchez <roberto@connexer.com>
--- libapache2-controller-perl-1.1.0.orig/t/conf/extra.last.conf.in
+++ libapache2-controller-perl-1.1.0/t/conf/extra.last.conf.in
@@ -2,6 +2,9 @@
 
 # @ServerRoot@ = t directory
 
+LoadModule mpm_worker_module /usr/lib/apache2/modules/mod_mpm_worker.so
+LoadModule authz_core_module /usr/lib/apache2/modules/mod_authz_core.so
+LoadModule authn_core_module /usr/lib/apache2/modules/mod_authn_core.so
 LoadModule apreq_module /usr/lib/apache2/modules/mod_apreq2.so
 
 # where Apache2::Controller and the test application libs can be found
